src/views/InteractiveMap/InteractiveMap.vue

```diff
@@ -457,59 +457,91 @@ export default Vue.extend({
           hasSimulationError: false
```

Comment:
There's unused `const operations = this.cardModifications(card);` above

Reply:
Removed!